### PR TITLE
Add React Static List of TODOs

### PR DESCRIPTION
**Pull request by ruslanpashkov**
[DEMO LINK](https://ruslanpashkov.github.io/react_static-list-of-todos/)

### CODE REVIEW

**Review by DidenkoDenis306 — APPROVED** [review]

File: src/components/TodoInfo/TodoInfo.tsx [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: FC<Props> = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article className={`TodoInfo ${completed && 'TodoInfo--completed'}`}>
```
Comment (33Orange) [inline src/components/TodoInfo/TodoInfo.tsx:13]: `completed && 'TodoInfo--completed'` this condition will be add 'false' as className if `completed === false
`

Try something else. 

image

**Review by 33Orange — CHANGES_REQUESTED** [review]
Good job. 

Just a one moment to fix

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1,4 +1,31 @@
...
+
+  return (
+    <article
+      className={cn(
+        'TodoInfo',
+        {
+          'TodoInfo--completed': completed,
+        },
+      )}
+    >
```
Comment (33Orange) [inline src/components/TodoInfo/TodoInfo.tsx:21]: Okay, it's interesting solution.
But to use a whole external library in order to insert 1 style ? :)
```suggestion
    <article className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`} >
```

**Review by 33Orange — APPROVED** [review]
Good job

**Pull request closed without merging**